label_studio/tasks/migrations/0054_add_brin_index_updated_at.py

```diff
@@ -1,30 +1,77 @@
 # Generated by Django on 2025-05-23 12:00
 
-from django.db import migrations, models
+from django.db import migrations, connection
 from django.conf import settings
-from django.contrib.postgres.indexes import BrinIndex
+from core.redis import start_job_async_or_sync
+from core.models import AsyncMigrationStatus
+import logging
 
-IS_SQLITE = settings.DJANGO_DB == settings.DJANGO_DB_SQLITE
+logger = logging.getLogger(__name__)
 
-if IS_SQLITE:
-    from django.db.migrations import AddIndex
-else:
-    from django.contrib.postgres.operations import AddIndexConcurrently as AddIndex
+migration_name = '0054_add_brin_index_updated_at'
 
+def forward_migration(migration_name):
+    migration = AsyncMigrationStatus.objects.create(
+        name=migration_name,
+        status=AsyncMigrationStatus.STATUS_STARTED,
+    )
+    logger.debug(f'Start async migration {migration_name}')
+    
+    # Check database backend and use appropriate SQL
+    if connection.vendor == 'postgresql':
```

Comment:
our standard pattern elsewhere has been to use IS_SQLITE, so `if not IS_SQLITE` would imply postgres